src/switchroot/ostree-prepare-root.c

```diff
@@ -570,7 +570,7 @@ main (int argc, char *argv[])
       else
         {
           /* Bind-mount /etc (at deploy path), and remount as writable. */
-          if (mount ("etc", TMP_SYSROOT "/etc", NULL, MS_BIND | MS_SILENT, NULL) < 0)
+          if (mount ("/etc", TMP_SYSROOT "/etc", NULL, MS_BIND | MS_SILENT, NULL) < 0)
```

Comment:
I'm not sure if this will actually resolve the coverity scan message.

Reply:
This is a bug, coverity is wrong here

Reply:
Will need to change this back.

If we insist on shutting up coverity here, this might work:

`#define TMP_SYSROOT_ETC TMP_SYSROOT "/etc"`

or

`static const char *tmp_sysroot_etc = TMP_SYSROOT "/etc"`

then just use that read-only global variable everywhere instead. The "static const char *" approach is probably slightly better, it's typesafe, but both are fine.